### DISCUSSION TITLE
KiCad 7.0 libraries

### DISCUSSION THREAD

**Discussion by ckirchhefer** [xesscorp/skidl#197] [Q&A]
Hey all,

I figured out a problem while trying to use the libraries of the current KiCad version.
First of all I have to add here that due to restrictions I'm not able / allowed to set environment variables so I add my paths within the Python script.
To do so, I `.append` strings to `lib_search_path` and `footprint_search_path` to set it to my local KiCad symbols / footprints folder.
First thing that came up, was a string-thing where `'..\KiCad\7.0\..'` was changed to `'..\KiCad\x07.0\\..'`. In order to fix this, I changed the string to `'..\\KiCad\\7.0\\..'` which seems to work as it appears correctly in the variable. 
But still, SkiDL gives me some errors where it appears that the KiCad 7 library is not working for some reason. To double-check if my `'\\'` string solution worked, I also tried using `'..\\KiCad\\6.0\\..'` to check if KiCad 6 libraries are working - and they do.

So is there a major mistake from my side or any trick to use the KiCad 7 libraries?

Thanks in advance and best regards

Christoph

**Comment by tapegoji** [comment]
I have the same issue. Were you able to fix it?
